src/main/kotlin/com.bugsnag.android.gradle/BugsnagPlugin.kt

```diff
@@ -42,31 +43,34 @@ class BugsnagPlugin : Plugin<Project> {
 
         project.afterEvaluate {
             // Make sure the android plugin has been applied first
-            if (project.plugins.hasPlugin(AppPlugin::class.java)) {
-                val android = project.extensions.getByType(AppExtension::class.java)
-                android.applicationVariants.all { variant ->
-                    applyBugsnagToVariant(variant, project, bugsnag, android)
-                }
-            } else {
+            if (!project.plugins.hasPlugin(AppPlugin::class.java)) {
                 throw IllegalStateException("Must apply \'com.android.application\' first!")
             }
+            if (!bugsnag.isEnabled) {
+                return@afterEvaluate
```

Comment:
Just so I understand what this is doing — this would be equivalent to a `continue` if this was a for loop ?

Reply:
This has the effect of skipping any further statements contained in the closure, i.e.: https://github.com/bugsnag/bugsnag-android-gradle-plugin/pull/227/files/f97634a605ce3259015c530ff23ce7b802f32bff#diff-ab1672558bc8b34166118b73667a1ca8R44